CHANGELOG.md

```diff
@@ -23,6 +23,12 @@ at anytime.
   *
   *
 
+
+
+## [0.10.0rc3] - 2017-04-21
+### Added
+  * Added functionality to check the balance of a given address.
```

Comment:
new changes should always go into the Unreleased section at the top. they'll get moved into the next release automatically when we create it